### PR TITLE
Change time widget unit format

### PR DESCRIPTION
**Pull request by PatrickCleary**
## Motivation

discussion on widget units. Still need to update tooltips to match.
https://transitmatters.slack.com/archives/GSJ6F35DW/p1687459523389939?thread_ts=1687459081.589399&cid=GSJ6F35DW

## Changes
Screenshot 2023-06-27 at 5 21 22 PM

## Testing Instructions

Look at all widget values please! Not sure I checked them all.

### CODE REVIEW

File: common/types/basicWidgets.tsx [hunk 1]
```diff
@@ -57,7 +59,12 @@ export class DeltaZonesWidgetValue extends BaseWidgetValue implements WidgetValu
   }
   getFormattedValue() {
     if (this.delta === undefined) return '...';
-    return `${this.delta > 0 ? '+' : '-'}${Math.abs(this.delta)}`;
+    return (
+      <p>
+        <WidgetText text={`${this.delta > 0 ? '+' : '-'}${Math.abs(this.delta)}`} />{' '}
```
Comment (mathcolo) [inline common/types/basicWidgets.tsx:64]: could you screenshot me what it looks like if it's positive?

Comment (PatrickCleary) [reply common/types/basicWidgets.tsx:64]: Screenshot 2023-06-27 at 5 28 27 PM

Comment (PatrickCleary) [reply common/types/basicWidgets.tsx]: Just realized it was showing a `-` for 0 change.

Changed it to 0 because I think `+0 zones` makes more sense

Comment (mathcolo) [reply common/types/basicWidgets.tsx]: you're killing me with that uppercase Z

but OK looks good! also unless you think -0 is a risk, you don't need the abs()

Comment (mathcolo) [reply common/types/basicWidgets.tsx]: oh yes I agree, +0 makes more sense indeed

**Review by mathcolo — APPROVED** [review]

File: common/types/basicWidgets.tsx (hunk 1 above)
Comment (PatrickCleary) [reply common/types/basicWidgets.tsx]: Screenshot 2023-06-27 at 5 40 20 PM

**Pull request merged**